### ISSUE TITLE
Docs incorrectly state the all boot settings will be displayed in the API

### ISSUE THREAD

**Issue by codewithcheese** [balena-io/docs#2081]
[Advanced boot settings](https://github.com/balena-io/docs/blob/dd73acb05ff4245ccb4265545f2e7e9578c8aca1/pages/reference/OS/advanced.md) states the following:

> Any values added to config.txt will be added to the {{ $names.cloud.lower }} API during device provisioning and displayed on the dashboard. 

However, `udev` rules are not displayed in the dashboard.

**Comment by vipulgupta2048** [comment]
Yeah, that doesn't sound right. @codewithcheese do make a PR to rectify this.
